### PR TITLE
Pixel precision

### LINKED ISSUE #1588: Incorrect position drift
**Describe the bug**
After the graph has been rotated, the position will be slightly offset when resized through the input box on the right, even if the rotation is adjusted back to 0, this problem persists(Then the program is likely to crash, it is not clear whether there is any connection with this BUG).However, shapes that are not rotated/or resized by dragging and dropping directly in the view will not be offset.

**To Reproduce**
1. Create an arbitrary graph
2. Rotate it
3. Use the scroll wheel in the input box to adjust the size
4. The graph moves a tiny distance to the right
5. Try to restore the rotation amount of the graph to 0, the problem still exists

**Expected behavior**
When I adjust the size, the position does not shift.

**Screenshots**

https://user-images.githubusercontent.com/40300919/153809822-bfa256ca-6493-47b6-a03d-4c52e8f42f17.mp4

**Desktop (please complete the following information):**

-   OS: Windows10
-   Browser:Microsoft Edge
-   Version 98.0.1108.50

**Environment (please complete the following information):**
SAAS (https://design.penpot.app)

### PR DESCRIPTION
**Pull request by Alotor**
https://tree.taiga.io/project/penpot/us/2552?milestone=312376

### CODE REVIEW

File: frontend/src/app/main/data/workspace/path/streams.cljs [hunk 1]
```diff
@@ -26,19 +27,38 @@
...
+      :else
+      (gpt/round position))))
+
 (defn drag-stream
   ([to-stream]
    (drag-stream to-stream (rx/empty)))
 
   ([to-stream not-drag-stream]
-   (let [start @ms/mouse-position
+   (let [
```
Comment (hirunatan) [inline frontend/src/app/main/data/workspace/path/streams.cljs]: join line

File: frontend/src/app/main/data/workspace/texts.cljs [hunk 2]
```diff
@@ -289,14 +289,6 @@
...
 ;; --- RESIZE UTILS
 
-(defn update-overflow-text [id value]
-  (ptk/reify ::update-overflow-text
-    ptk/UpdateEvent
-    (update [_ state]
-      (let [page-id (:current-page-id state)]
-        (update-in state [:workspace-data :pages-index page-id :objects id] assoc :overflow-text value)))))
-
-
```
Comment (hirunatan) [inline frontend/src/app/main/data/workspace/texts.cljs:299]: Why is this removed? Perhaps because text overflow cannot occur now?

File: frontend/src/app/util/data.cljs [hunk 3]
```diff
@@ -168,3 +168,10 @@
           nm (str/trim (str/lower name))]
       (str/includes? nm st))))
 
+(defn tap
+  "Works like rx/tap but for normal collections"
+  ;; Signature for transducer use
+  ([f]
+   (map #(do (f %) %)))
+  ([f coll]
+   (map #(do (f %) %) coll)))
```
Comment (hirunatan) [inline frontend/src/app/util/data.cljs]: In `app.common.data` there are tap and tap-r functions, I think would be good to unify them.

File: frontend/src/app/main/data/workspace/texts.cljs (hunk 2 above)
Comment (Alotor) [reply frontend/src/app/main/data/workspace/texts.cljs:299]: Yes, the text overflowed is now displayed. This is something that I changed after talking to the designers.

File: frontend/src/app/util/data.cljs (hunk 3 above)
Comment (Alotor) [reply frontend/src/app/util/data.cljs]: You're right. I'll remove mine, it's redundant
